_ext/css_minifier.rb

```diff
@@ -58,26 +58,9 @@ def transform(site, page, input)
           return input
         end
 
-        oldFileName = File.basename(page.output_path).to_s
-
-        # Create new file name with suffix added
-        newFileName = oldFileName.slice(0..oldFileName.length-4)+"min.css"
-        newOutputPath = File.join(File.dirname(page.output_path.to_s),newFileName)
-
-        # Create a temporary file with the merged content.
-        tmpOutputPath = File.join( "./_tmp/" , newFileName)
-        tmpOutputFile = File.new(tmpOutputPath,"w")
-        tmpOutputFile.write(output)
-        tmpOutputFile.close
-
-        # Add the temporary file to the list of pages for rendering phase.
-        newPage = site.engine.load_page(tmpOutputPath)
-        newPage.source_path = tmpOutputPath
-        newPage.output_path = newOutputPath
-        site.pages << newPage
-
+        page.output_path = page.output_path.gsub('.css', '.min.css')
         # We return the input because we leave the original file untouched
-        input
+        output
```

Comment:
No, it won't work in 0.5.5, this will be for 0.5.6 and moving forward.

It'll keep the css name if the minification transformers are not used.

On Tuesday, April 28, 2015, Max Rydahl Andersen <notifications@github.com>
wrote:

> In _ext/css_minifier.rb
> <https://github.com/maxandersen/jbosstools-website/pull/5#discussion_r29245567>
> :
>
> >          # We return the input because we leave the original file untouched
> > -        input
> > +        output
>
> so its renaming its output and this would work with both old and new
> awestruct ?
>
> and would it still leave the .css in place ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maxandersen/jbosstools-website/pull/5/files#r29245567>
> .
>


-- 
Jason Porter
http://en.gravatar.com/lightguardjp